src/Jobs/Consumer/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace Byndyusoft.Dotnet.Core.Samples.Jobs.Consumer
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using Autofac;
+    using Autofac.Extensions.DependencyInjection;
+    using Infrastructure.Dapper.ConnectionsFactory;
+    using Installers;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using NLog.Extensions.Logging;
+    using Workers;
+
+    class Program
+    {
+        private static IConfigurationRoot _configuration;
+        private static IContainer _container;
+
+        private static ILoggerFactory _loggerFactory;
+
+        static void Main(string[] args)
+        {
+            BuildConfiguration();
+            ConfigureDependencies();
+            ConfigureLogging();
+
+            _container.Resolve<ConsumerWorker>().Start();
+        }
+
+        private static void ConfigureLogging()
+        {
+            _loggerFactory
+                .AddNLog()
+                .AddConsole(_configuration.GetSection("Logging"))
+                ;
+        }
+
+        private static void BuildConfiguration()
+        {
+            var environment = Environment.GetEnvironmentVariable("SERVICE_ENVIRONMENT");
```

Comment:
We can use fragment like 
```
var environmentConfiguration = new ConfigurationBuilder()
   .AddCommandLine(commandLineArguments)
   .AddEnvironmentVariables("SERVICE_")
   .Build();
var environmentName = environmentConfiguration["environment"];
```
for getting settings from environment variables and command line, command line seems to me easier than environment variables.

Reply:
done: https://github.com/Byndyusoft/Byndyusoft.Dotnet.Core.Infrastructure/pull/11/commits/9263f97aa30a4de3a0a6539ca555bea0fd9cd179